### PR TITLE
Serathil for Kash'drakor

### PR DESCRIPTION
**Pull request by arithon**
## Changelog:
As requested, gave Serathil to its initial owner in the first start date

### CODE REVIEW

**Review by LordSilvermane — APPROVED** [review]
Works fine with the fix!

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 1]
```diff
@@ -991,21 +991,23 @@ legion_doomhorn = {
...
-			NOT = { year = 605 }
-		}
 	}
 	
 	weight = {
 		value = 0
 		additive_modifier = {
 			value = 100
 			character = 52058						# Kor'Gall Stonemaul
+			year = 584
```
Comment (zumbak04) [inline common/artifact_spawns/wc_artifact_spawns.txt:1005]: After the year 584, both characters have the same chance to get Serathil. `year = 584` means equal or greater than the year 584.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/artifact_spawns/wc_artifact_spawns.txt (hunk 1 above)
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt:1005]: Meme said that as well, but Kash'drakor dies before the next start date

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**